src/bindings/jna/README.md

```diff
@@ -23,10 +23,18 @@ If this fails, make sure you have at least Java version 11 or higher installed a
 
 To use the bindings in a Java project, we have to include the jar file
 libelektra-\$VERSION.jar in the project. The version number is the same
-one as used for Elektra. This jar is created upon build of Elektra if
-you enable the JNA bindings, e.g., with `cmake -DBINDINGS=jna`, see also
-[COMPILE](/doc/COMPILE.md#bindings). Internally, Gradle will be used to
-actually compile the plugin.
+one as used for Elektra.
+
+### Build
+
+This jar is created upon building El of Elektra if you inclued the JNA bindings dring the build process, e.g., with `cmake -DBINDINGS=jna`. Check out [COMPILE](/doc/COMPILE.md#bindings) for more details on how to include bindings in the build process.
```

Comment:
We have some typos here. I suggest changing that sentence to
`This jar is created upon building Elektra, if you included the JNA bindings during the build process`

Reply:
Not quite sure what happened there 😅. It's fixed now.